grpc/interop/README.md

```diff
@@ -0,0 +1,61 @@
+grpc-finagle interop client/server
+===============================
+
+In order to test how well grpc-finagle interoperates with other grpc stacks, we
+implement the [https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md](standard interop service).
+
+## Current status
+
+Some of the basic interop tests work, but many need improvements.
+
+## Test implementation status
+
+These test names are standard and defined in the above document.
+
+- [x] `empty_unary` empty (zero bytes) request and response.
+- [ ] `large_unary` single request and (large) response. (flow control deadlock) #929
+- [ ] `client_streaming` request streaming with single response. (flow control deadlock) #982
+- [x] `server_streaming` single request with response streaming.
+- [ ] `ping_pong` full-duplex streaming. (flow control deadlock) #980
+- [x] `empty_stream` full-duplex streaming with zero message. #981
+- [x] `timeout_on_sleeping_server` fullduplex streaming on a sleeping server.
```

Comment:
just noting that our test client is not yet able to enforce deadlines on streams (needs API changes)